specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md

```diff
@@ -34,3 +34,12 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'runtime_3_0_preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/cognitiveservices/v3.0/luis/$(namespace)
 ```
+
+### Tag: runtime_3_0 and go
```

Comment:
Remove the preview references before adding this.

Reply:
@moabba you mean all the preview here? or do i just fix the 'preview' in the v3.0 part?

Reply:
Everywhere in this files and the other files. Also delete the preview folder altogether.